lib/rig_inbound_gateway/api_proxy/handler/http.ex

```diff
@@ -116,7 +116,7 @@ defmodule RigInboundGateway.ApiProxy.Handler.Http do
         |> with_cors()
         |> Tracing.Plug.put_resp_header(Tracing.context())
         |> Conn.put_resp_content_type("application/json")
-        |> Conn.send_resp(:ok, response)
+        |> Conn.send_resp(response_code, response)
```

Comment:
what about the content type? Ideally, we'd forward the body as-is and also retain the content-type, then the backend can send whatever